draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -805,7 +805,7 @@ and variant then replace the respective bits as defined by {{version_field}}
 and {{variant_field}},
 
 Alternatively, an implementation MAY choose to randomly generate the exact required number of bits for
-for random_a, random_b, and random_c then concatenate the version and variant in the required position.
+random_a, random_b, and random_c (122 bits total) then concatenate the version and variant in the required position.
```

Comment:
We might as well include the exact number of bits here - 122 - since we mention it.